src/MCPClient/lib/settings/common.py

```diff
@@ -16,20 +16,20 @@
 # along with Archivematica.  If not, see <http://www.gnu.org/licenses/>.
 
 import os, sys, ConfigParser
-import django_mysqlpool
 
 # Get DB settings from main configuration file
 config = ConfigParser.SafeConfigParser()
 config.read('/etc/archivematica/archivematicaCommon/dbsettings')
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django_mysqlpool.backends.mysqlpool',
+        'ENGINE': 'django.db.backends.mysql',
         'NAME': config.get('client', 'database'),          # Or path to database file if using sqlite3.
         'USER': config.get('client', 'user'),              # Not used with sqlite3.
         'PASSWORD': config.get('client', 'password'),      # Not used with sqlite3.
         'HOST': config.get('client', 'host'),              # Set to empty string for localhost. Not used with sqlite3.
         'PORT': '',                                        # Set to empty string for default. Not used with sqlite3.
+        'CONN_MAX_AGE': 500,
```

Comment:
Do we want to set this to `None`, which allows unlimited persistent connections? https://docs.djangoproject.com/en/1.8/ref/settings/#std:setting-CONN_MAX_AGE